app/ui/src/app/connections/common/configuration/configuration.service.ts

```diff
@@ -44,6 +44,9 @@ export class ConnectionConfigurationService {
       case 'activemq':
         controls = ['brokerUrl', 'username', 'password', 'clientId', 'skipCertificateCheck', 'brokerCertificate', 'clientCertificate'];
         break;
+      case 'aws-s3':
+        controls = ['accessKey', 'secretKey', 'region', 'bucketNameOrArn'];
```

Comment:
I think this is fine for now, but we should get this field set at the backend, created https://github.com/syndesisio/syndesis/issues/1204

Reply:
Yeah, absolutely. It's the only way I know/found for the moment :-)

Reply:
No worries, should have raised that issue earlier :-)